conf/modules.config

```diff
@@ -129,46 +132,48 @@ process {
                 (params.fdr_level != 'psm-level-fdrs') ? "-" + params.fdr_level : ""
```

Comment:
Not related to the PR, but in general: Why do we use 5% FDR in Percolator but we are filtering with 1% FDR in OPENMS_IDFILTER_FOR_ALIGNMENT and OPENMS_IDFILTER_Q_VALUE